Tools/CISupport/ews-build/steps.py

```diff
@@ -3123,15 +3123,17 @@ def parseOutputLine(self, line):
             self.error_context_buffer = []
 
 
-class CompileWebKit(shell.Compile, AddToLogMixin):
+class CompileWebKit(shell.Compile, AddToLogMixin, ShellMixin):
     name = 'compile-webkit'
     description = ['compiling']
     descriptionDone = ['Compiled WebKit']
     env = {'MFLAGS': ''}
     warningPattern = '.*arning: .*'
     haltOnFailure = False
-    command = ['perl', 'Tools/Scripts/build-webkit', WithProperties('--%(configuration)s')]
+    build_command = ['perl', 'Tools/Scripts/build-webkit', WithProperties('--%(configuration)s')]
+    filter_command = ['perl', 'Tools/Scripts/filter-build-webkit', '-logfile', 'WebKitBuild/build-log.txt']
     VALID_ADDITIONAL_ARGUMENTS_LIST = []  # If additionalArguments is added to config.json for CompileWebKit step, it should be added here as well.
+    APPLE_PLATFORMS = ('mac', 'ios', 'tvos', 'watchos')
```

Comment:
It might be better to make a list of Non-Apple platforms since that list will be smaller and less likely to change.

Also, this excludes queues with platform as '*', which is fine currently, but in future it might change.